lib/miquido_lints.yaml

```diff
@@ -1,11 +1,6 @@
 analyzer:
   plugins:
     - dart_code_metrics
-
-  # Explanation for this settings are described in Step 3 of this https://dash-overflow.net/articles/getting_started/
-  strong-mode:
-    implicit-casts: false
```

Comment:
Hmmm... I know that by default those values are true but I would recommend leave it and just in comment put link to https://dart.dev/resources/dart-3-migration#analyzer-config

Reply:
Removing those rules is necessary because they're no longer valid in the `analysis_options` YAML structure. Keeping them here leads to the `flutter analyze` command returning issues.

Reply:
Sorry, my bad, I meant to set those values to true:
![image](https://github.com/miquido/lint_quido/assets/17902851/cec1a421-0b73-451c-85df-ac5431d1b1ee)

meaning replace those with
```yaml
  language:
    strict-casts: true
    strict-raw-types: true
  ```

Reply:
This has already been done in one of the previous PRs, it's set below the `exclude` section:
<img width="466" alt="Zrzut ekranu 2023-05-12 o 08 45 59" src="https://github.com/miquido/lint_quido/assets/1737922/927672da-2dc6-49b3-8be2-ecd388ecf758">